### PR TITLE
Update overclocking.md

### PR DESCRIPTION
**Pull request by generaleramon**
i started to add some infos about the 4B

### CODE REVIEW

File: configuration/config-txt/overclocking.md [hunk 1]
```diff
@@ -11,8 +11,8 @@ Overclocking and overvoltage will be disabled at runtime when the SoC reaches 85
...
+| gpu_freq | Sets `core_freq`, `h264_freq`, `isp_freq`, and `v3d_freq` together; on Pi 1/Pi 2, the default value is `250` for all items, on Pi 3/Pi Zero /Pi Zero W `core_freq` defaults to `400`, while `h264_freq`, `isp_freq`, and `v3d_freq`default to `300`; on the Pi 4B the `core_freq`, `h264_freq`, `isp_freq`, and `v3d_freq` default to `500`, `600` is the only other accepted value|
+| core_freq | Frequency of the GPU processor core in MHz, influences CPU performance because it drives the L2 cache and memory bus; default value is `250` for the Pi 1/Pi 2, `400` for the Pi 3/Pi Zero/Pi Zero W. For the Pi 4B the default value is `500`, `600` is the only other accepted value; the L2 cache benefits only the Pi Zero/Pi Zero W/ Pi 1, there is a small benefit for SDRAM on the Pi 2/Pi 3, and there is no effect on the SDRAM on the Pi 4B|
```
Comment (generaleramon) [inline configuration/config-txt/overclocking.md]: https://github.com/raspberrypi/documentation/issues/1202#issuecomment-512253438

**Comment by JamesH65** [comment]
@timg236 @dom Are these numbers accurate for the Pi4?

**Comment by lurch** [comment]
@JamesH65 Dom's GitHub handle is @popcornmix :wink:

**Comment by JamesH65** [comment]
Bugger, that's his gitlab handle. Too many repos!

**Comment by timg236** [comment]
I think the numbers are ok for the current firmware. The core-clock is not just the VPU and the firmware will actively change it to optimise power/performance and this will change in future releases.
For over-clocking setting core_freq_min=core_freq should be accepted but that's untested and might result in display issues.

File: configuration/config-txt/overclocking.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Overclocking and overvoltage will be disabled at runtime when the SoC reaches 85
...
-| core_freq | Frequency of the GPU processor core in MHz, influences CPU performance because it drives the L2 cache and memory bus; default value is `250` for the Pi 1/Pi 2, `400` for the Pi 3/Pi Zero/Pi Zero W. For the Pi 4B the default value is `500`, `600` is the only other accepted value; the L2 cache benefits only the Pi Zero/Pi Zero W/ Pi 1, there is a small benefit for SDRAM on the Pi 2/Pi 3, and there is no effect on the SDRAM on the Pi 4B|
+| core_freq | Frequency of the GPU processor core in MHz, influences CPU performance because it drives the L2 cache and memory bus; default value is `250` for the Pi 1/Pi 2 and `400` for the Pi 3/Pi Zero/Pi Zero W. For the Pi 4B the default value is `500`, `600` is the only other accepted value; the L2 cache benefits only the Pi Zero/Pi Zero W/ Pi 1, there is a small benefit for SDRAM on the Pi 2/Pi 3, and there is no effect on the SDRAM on the Pi 4B|
```
Comment (popcornmix) [inline configuration/config-txt/overclocking.md]: This isn't correct. core_freq affects sdram bandwidth on a Pi4 just like the Pi2/Pi3.

**Comment by popcornmix** [comment]
Looks reasonable, but this info is subject to change.
On Pi4 gpu_freq_min will probably be reduced, and restrictions on core_freq may be relaxed, but info can be updated when it is finalised.

**Comment by andrum99** [comment]
Further to popcornmix's comments, you may want a note somewhere that Pi 4 overclocking options may be subject to change in the future as the platform matures.

File: configuration/config-txt/overclocking.md (hunk 2 above)
Comment (generaleramon) [reply configuration/config-txt/overclocking.md]: Fixed in the latest edit

**Comment by JamesH65** [comment]
@popcornmix Are these numbers still correct? Does temp_soft_limit, if defined, work on the Pi4 but dropping to 1000?

**Comment by popcornmix** [comment]
temp_soft_limit does nothing on Pi4 currently.

**Comment by JamesH65** [comment]
OK, so this PR is incorrect at least in that respect. @generaleramon

**Comment by JamesH65** [comment]
@generaleramon Can you update to include the changes required above, then I can push to copy edit.

**Comment by generaleramon** [comment]
> @generaleramon Can you update to include the changes required above, then I can push to copy edit.

i think everything is fixed/included.

**Comment by JamesH65** [comment]
Great, flagged up for copy/edit.

**Pull request merged**